src/psm/src/pdnsim.tcl

```diff
@@ -226,6 +226,25 @@ proc set_pdnsim_net_voltage { args } {
   }
 }
 
+sta::define_cmd_args "set_pdnsim_inst_power" {
+  -inst instance
+  -power power
+  [-corner corner]}
+
+proc set_pdnsim_inst_power { args } {
+  sta::parse_key_args "set_pdnsim_inst_power" args \
+    keys {-inst -corner -power} flags {}
+  if { [info exists keys(-inst)] && [info exists keys(-power)] } {
+    set inst [psm::find_inst $keys(-inst)]
+    set power $keys(-power)
+    set corner [sta::parse_corner_or_all keys]
+    psm::set_inst_power $inst $corner $power
```

Comment:
What if the instance or corner are not found?

Reply:
instance not found is handled in `psm::find_inst` and corner is handled in `sta::parse_corner_or_all`